packages/chip/src/Chip/Chip.styles.ts

```diff
@@ -39,175 +46,270 @@ export const chipWrapperPadding: Record<
 > = {
   [BaseFontSize.Body1]: {
     y: 0,
-    x: 6,
+    x: 4,
   },
   [BaseFontSize.Body2]: {
     y: 2,
-    x: 6,
+    x: 4,
   },
 } as const;
 
-/**
- * Chip Variants
- */
-export const variantColor: Record<
-  Variant,
-  Record<
-    Theme,
-    {
-      color: string;
-      bgColor: string;
-      focusBgColor: string;
-    }
-  >
-> = {
+export const variantColor = {
```

Comment:
does it make sense to use color tokens for these values? if we don't know if there are intentional differences, are there gaps we should surface to design?

Reply:
As in create new tokens in `Tokens` for chip?

Reply:
I just spoke with Rob, and the current tokens are almost exclusively for input which chips are not a part of. We don't have component-specific tokens yet.